resources/assets/sass/_github.scss

```diff
@@ -0,0 +1,89 @@
+#github-login-panel {
+	text-align: center;
+}
+
+#github-login-panel .btn {
+	margin-bottom: 0px;
```

Comment:
A few notes about CSS styling for this file:

1. CSS properties should be alphabetized.
2. Properties should also be indented by 4 spaces.
3. There should be a newline after each set.